### PR TITLE
Handle non int/long values in coverage reports

### PR DESCRIPTION
**Pull request by kageiit**
Some tools output coverage numbers like "1.0" instead of "1". This behavior is not usual, but the plugin should be able to handle those

### CODE REVIEW

File: src/main/java/com/uber/jenkins/phabricator/coverage/XmlCoverageProvider.java [hunk 1]
```diff
@@ -112,6 +112,14 @@ private void parse(Set<String> includeFiles, Set<File> reports) throws SAXExcept
         }
     }
 
+    private static Long getLongValue(NamedNodeMap attrs, String attr) {
+        return Math.round(Double.valueOf(attrs.getNamedItem(attr).getTextContent()));
```
Comment (edsilfer) [inline src/main/java/com/uber/jenkins/phabricator/coverage/XmlCoverageProvider.java]: Should we care for `NumberFormatException` in here?

File: src/main/java/com/uber/jenkins/phabricator/coverage/XmlCoverageProvider.java [hunk 2]
```diff
@@ -112,6 +112,14 @@ private void parse(Set<String> includeFiles, Set<File> reports) throws SAXExcept
...
+    }
+
+    private static Integer getIntValue(NamedNodeMap attrs, String attr) {
+        return Math.round(Float.valueOf(attrs.getNamedItem(attr).getTextContent()));
```
Comment (edsilfer) [inline src/main/java/com/uber/jenkins/phabricator/coverage/XmlCoverageProvider.java]: Same here

Comment (kageiit) [reply src/main/java/com/uber/jenkins/phabricator/coverage/XmlCoverageProvider.java]: This change was to fix numberformat exception. I can make the error clearer

Comment (edsilfer) [reply src/main/java/com/uber/jenkins/phabricator/coverage/XmlCoverageProvider.java]: :+1:

**Pull request merged**